styles/dcc.scss

```diff
@@ -31,7 +31,7 @@ $duvall: "DuvallSmallCaps", "Palatino Linotype", serif;
 }
 
 .dcc .flex-padding {
-  flex: 1 0 0px;
```

Comment:
Why is the autoformatter changing units?

Reply:
More worryingly, sometimes it's removed the units and in other places they're being added.

Reply:
Yep, those were changes I made manually that are there to keep the CSS linter happy.  Here’s that rule: https://github.com/CSSLint/csslint/wiki/Disallow-units-for-zero-values

There was also one where it was complaining because the “px” was missing off of a flex, so I added it there.